adapters/oidc/js/src/main/resources/keycloak.d.ts

```diff
@@ -271,15 +279,14 @@ declare namespace Keycloak {
 	 * Conditional CompatPromise type in order to support
 	 * both legacy promises and native promises as return types.
 	 */
-	type PromiseType = KeycloakPromiseType | undefined;

```

Comment:
This extra type has been removed as `'legacy'` is now passed everywhere explicitly.